packages/logger/src/app-insights-logger-client.spec.ts

```diff
@@ -75,16 +75,30 @@ describe(AppInsightsLoggerClient, () => {
     });
 
     describe('trackMetric', () => {
-        it('when value passed', async () => {
+        it('when properties not passed', async () => {
+            setupCallsForTelemetrySetup();
+            await testSubject.setup(null);
+            appInsightsTelemetryClientMock.reset();
+
+            appInsightsTelemetryClientMock
+                .setup(t => t.trackMetric(It.isValue({ name: 'QueuedScanRequests', value: 10, properties: {} })))
+                .verifiable();
+
+            testSubject.trackMetric('QueuedScanRequests', 10);
+
+            verifyMocks();
+        });
+
+        it('when properties passed', async () => {
             setupCallsForTelemetrySetup();
             await testSubject.setup(null);
             appInsightsTelemetryClientMock.reset();
 
             appInsightsTelemetryClientMock
-                .setup(t => t.trackMetric(It.isValue({ name: 'metric1', value: 10, properties: {} })))
+                .setup(t => t.trackMetric(It.isValue({ name: 'QueuedScanRequests', value: 1, properties: { foo: 'bar' } })))
```

Comment:
NIT: extract value and properties to a variable could help if there are changes to the method in the future.